rero_ils/query.py

```diff
@@ -387,14 +390,14 @@ def _default_parser(qstr=None, query_boosting=None):
             qstr = _PUNCTUATION_REGEX.sub(' ', qstr)
             qstr = re.sub(r'\s+', ' ', qstr).rstrip()
             if not query_boosting:
-                return Q(query_type, query=qstr,
+                return Q(query_type, lenient=lenient, query=qstr,
                          default_operator=default_operator)
             else:
                 return Q('bool', should=[
```

Comment:
Note (to be tested) : instead of `Q('bool', should=[Q(), Q(), ...]` we could use simpler notation `return Q() | Q()`